### PR TITLE
Text review

### PR DESCRIPTION
**Pull request by krisztinaknagy**
Simplifying some text elements.

### CODE REVIEW

**Comment by nmldiegues** [comment]
@krisztinaknagy please consider improving this with, e.g., a README explaining how to use the generic R provider. We have a basic README for that on the python counter-part (as discussed in https://github.com/feedzai/feedzai-openml-python/pull/5).

@pedrorijo91 will be available to help you with any doubt, as he is preparing materials also to present this on next wednesday.

File: README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ using [RServe](https://www.rforge.net/Rserve/doc.html).
 ### Generic R
 [![Maven metadata URI](https://img.shields.io/maven-metadata/v/http/central.maven.org/maven2/com/feedzai/openml-generic-r/maven-metadata.xml.svg)](https://mvnrepository.com/artifact/com.feedzai/openml-generic-r)
 
-You can find in the `openml-generic-r` module a provider that allows
-users to load in code that conforms to a simple API.
+The `openml-generic-r` module contains a provider that allows
+developers to load code that conforms to a simple API.
```
Comment (pedrorijo91) [inline README.md]: maybe update to `developers to load R code` ? It's implicit that the code loaded should be in the R programming language, but better explicit than implicit, do you agree?

File: README.md [hunk 2]
```diff
@@ -44,19 +44,19 @@ This module can be pulled from Maven Central:
...
 mvn clean install
 ```
 
 
 ## Prerequisites for running tests
 
-To use these providers you need to have [R Project](https://www.r-project.org/) installed on your environment.
-After installing R, you need to install the R packages that are being used by the provider. The easiest way is to install them from CRAN.
+To use these providers you need to have [R Project](https://www.r-project.org/) installed in your environment.
+After installing R, you need to install the R packages that the provider uses. The easiest way is to install them from CRAN.
```
Comment (pedrorijo91) [inline README.md]: should we add a CRAN link to https://cran.r-project.org/ ?

**Comment by nmldiegues** [comment]
I don't mind that approach @pedrorijo91 
Whatever you guys do, remember to not just land on develop, but also cherry-pick to hf-0.1.X (a developer should do it, e.g., either you @pedrorijo91  or me).

**Comment by pedrorijo91** [comment]
cherry picked to hf-0.1.X

**Pull request merged**